maven-plugin/src/main/java/org/openmrs/maven/plugins/utility/SpaInstaller.java

```diff
@@ -54,12 +54,12 @@ public void installFromDistroProperties(File serverDir, DistroProperties distroP
         Map<String, String> spaProperties = distroProperties.getSpaProperties(distroHelper, serverDir);
         if (!spaProperties.isEmpty()) {
             JsonObject spaConfigJson = convertPropertiesToJSON(spaProperties);
-            File spaConfigFile = new File(serverDir, "microfrontends.json");
+            File spaConfigFile = new File(serverDir, "spa-build-config.json");
             writeJSONObject(spaConfigFile, spaConfigJson);
             nodeHelper.installNodeAndNpm(NODE_VERSION, NPM_VERSION);
             File buildTargetDir = new File(serverDir, BUILD_TARGET_DIR);
-            nodeHelper.runNpx(String.format("openmrs@latest build --target %s", buildTargetDir));
-            nodeHelper.runNpx(String.format("openmrs@latest assemble --target %s --mode config --config %s", buildTargetDir, spaConfigFile));
+            nodeHelper.runNpx(String.format("openmrs@next build --target %s --build-config %s", buildTargetDir, spaConfigFile));
+            nodeHelper.runNpx(String.format("openmrs@next assemble --target %s --mode config --config %s", buildTargetDir, spaConfigFile));
```

Comment:
Future work: we should probably make which version of the openmrs tool to use configurable via a Maven parameter.